scripts/splunk-to-segment.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# splunk-to-segment.sh
+#   Adapt user journey recordes loaded from Splunk for uploading into Segmment:
+#   - Map cluster usernames to SSO user IDs
+#   - Convert nested JSON objects from strings to actual objects.
```

Comment:
This is mentioned on the first script as well, is the conversion done twice? I don't see this script using `fetch-uj-records.sh`, or the opposite

Reply:
This script implements the things that `fecth-uj-records.sh` specifies that need to be implemented, the scripts do not use each other, they are meant to be piped together.